### ISSUE TITLE
[Feature Request] auto delete closed tasks

### ISSUE THREAD

**Issue by ramack** [dmfs/opentasks#605]
It would be great to have a setting to automatically delete tasks after they have been marked "done" (closed, finished) for some configurable time.

**Comment by dmfs** [comment]
Right, that shouldn't be difficult to do. In the end it's just a setting of a number of days and a job which runs once per day to remove all completed tasks which have been completed at least X days before.

**Comment by dmfs** [comment]
I think we can squeeze this one in after initial support for recurrence has been completed.

**Comment by hg2581** [comment]
I most often deleted tasks I have finished but there are also tasks that I definitely need to save even after finishing them. Important not to lose finished tasks by accident!

**Comment by dmfs** [comment]
Right, so the setting should make clear that tasks are irrevocably deleted when enabled. Maybe with an additional confirmation step that can't be skipped accidentally, like a check box that must be ticked before you can proceed.

**Comment by Ronkn** [comment]
Looks like this may have fizzled out. I have 2 years of completed tasks that I'd rather not have to delete manually. Any chance this is still on the radar to implement?
